COMMON_ISSUES.md

```diff
@@ -48,9 +48,42 @@ policies here:
 - [Firefox Autoplay Policy](https://hacks.mozilla.org/2019/02/firefox-66-to-block-automatically-playing-audible-video-and-audio/)
 - [Safari Autoplay Policy](https://webkit.org/blog/7734/auto-play-policy-changes-for-macos/)
 
-Playback of RemoteAudioTracks should not be affected since its playback does not
-depend on the \<audio\> element. However, for RemoteVideoTracks, there are two ways
-to ensure playback:
+Playback of RemoteAudioTracks should not be affected in Chrome and Firefox. Safari will
+pause \<audio\> elements that play back RemoteAudioTracks if no local media is being captured.
+They can be played by the application after a user interaction.
```

Comment:
I think instead of the sample code to determine if `isUserInteractionRequired` we should  just suggest that they add a button to start playback.

Reply:
@makarandp0 I think the programmatic approach is better because tracks might be acquired after some remote tracks are subscribed to, so later subscriptions need not have the playback button.